### PR TITLE
staging-API beging used for dev

### PR DESCRIPTION
**Pull request by Pavangbhat**
closes issue #121

### CODE REVIEW

File: src/task/script.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-const BASE_URL = 'https://api.realdevsquad.com';
+const BASE_URL = window.BASE_URL;
```
Comment (swarajpure) [inline src/task/script.js:1]: BASE URL for what? What if we call it `API_BASE_URL`?

File: src/helpers/loadENV.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+window.location.hostname !== 'dashboard.realdevsquad.com'
+  ? (window.BASE_URL = 'https://staging-api.realdevsquad.com')
+  : (window.BASE_URL = 'https://api.realdevsquad.com');
```
Comment (swarajpure) [inline src/helpers/loadENV.js:3]: Sweet!

File: package.json [hunk 3]
```diff
@@ -6,8 +6,7 @@
   "scripts": {
     "check": "prettier --check .",
     "fix": "prettier --write .",
-    "start": "node server.js",
-    "dev": "nodemon server.js"
+    "start": "node server.js"
```
Comment (swarajpure) [inline package.json:9]: Why are we removing nodemon?

File: src/task/script.js (hunk 1 above)
Comment (Pavangbhat) [reply src/task/script.js:1]: Right 👍

File: package.json (hunk 3 above)
Comment (Pavangbhat) [reply package.json:9]: Any changes made in any file will restart the server but does not refresh the page

Comment (swarajpure) [reply package.json:9]: makes sense

File: src/helpers/loadENV.js [hunk 4]
```diff
@@ -0,0 +1,3 @@
+window.location.hostname !== 'dashboard.realdevsquad.com'
+  ? (window.API_BASE_URL = 'https://staging-api.realdevsquad.com')
+  : (window.API_BASE_URL = 'https://api.realdevsquad.com');
```
Comment (swarajpure) [inline src/helpers/loadENV.js:3]: @akshay1502 does this piece of code ring some bell for some feature? 😄

Comment (akshay1502) [reply src/helpers/loadENV.js:3]: Yup. For staging and production login right?

**Pull request merged**